benchmarks/docker/docker_build.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Builds a docker image nighthawk-dev:latest containing the stripped binaries
+# based on a pre-build bazel-bin directory (with "-c opt" set).
+
+# Stop on errors.
+set -e
+
+# NOTE: explicit no -x for verbose commands. Because this is run in CI, doing so may result in
+# publishing sensitive information into public CI logs if someone makes a change in a
+# consuming script that is off guard.
+
+DOCKER_NAME="benchmark"
+DOCKER_IMAGE_PREFIX="oschaaf/${DOCKER_NAME}"
+BAZEL_BIN="$(bazel info bazel-bin)"
+WORKSPACE="$(bazel info workspace)"
+bazel build //benchmarks:benchmarks
+TMP_DIR="${WORKSPACE}/tmp-docker-build-context"
```

Comment:
Why not use `mkdtemp`? Then you don't have to `rm -rf` in the next line.

Reply:
I think `mkdtemp` is a python thing, whereas this is bash? At least my system can't find that binary :-)